webpack.config.js

```diff
@@ -345,7 +345,7 @@ function get_msggrep_options () {
 }
 
 const plugins = [
-    new IncludedModulesPlugin((section || "") + "included-modules"),
+    production ? new IncludedModulesPlugin((section || "") + "included-modules") : undefined,
```

Comment:
I'm not sure about that -- Isn't that required for developers for correct `make` dependencies? Or does this *only* affect dependencies to `node_modules`, not to e.g. pkg/lib/?

Can you please add a rationale to the commit message?

Reply:
Let me move this commit outside of this PR as it's not so relevant.